coss/home/templates/home/about_page.html

```diff
@@ -15,46 +15,80 @@
   <div class="content">
 
     <nav id="nav-about" class="nav nav-about justify-content-center">
-      {% if page.about.body_text %}
+      {% if page.about.body_title %}
         <a class="nav-link" href="#body">{{ page.about.body_title }}</a>
       {% endif %}
-      {% if page.about.info_text %}
+      {% if page.about.info_title %}
         <a class="nav-link" href="#info">{{ page.about.info_title }}</a>
       {% endif %}
-      {% block extra_about_nav %}
-      {% endblock %}
+      {% if page.about.details_title %}
+        <a class="nav-link" href="#details">{{ page.about.details_title }}</a>
+      {% endif %}
     </nav>
 
     {% if page.about.body_text %}
-      <div id="body" class="body">
+      {% if page.about.body_image %}
+        {% image page.about.body_image original as bg_photo %}
+      {% endif %}
+      <div id="body" class="body" data-bg-img="{{ bg_photo.url }}">
+        <div class="body-image"></div>
         <div class="well">
+          {% if page.about.body_title %}
```

Comment:
Shouldn't this `if` be before the `<div class="well"`

Reply:
Not necessarily. We won't to display the well and `page.about.body_text` even when no title is set.

Reply:
Agreed, but the comment stands for the body_text too. It's a not required field. Ideally, as we make the project more generic, we want to only render blocks that have content. In the future, this can be a standalone snippet inserted in a template.
Probably, it's OK for now, but please add at least comments as placeholders on what we need to revisit in order to make the template more generic.

Reply:
Yes, adding comments is a good idea. Just to point out that this whole block is wrapped inside an if statement (`{% if page.about.body_text %}`), so this is rendered only if a body text is provided.